dist-persist/wbstack/src/Settings/LocalSettings.php

```diff
@@ -52,44 +52,24 @@
     $wgMWLoggerDefaultSpi = [
         'class' => \WBStack\Logging\CustomSpi::class,
         'args' => [[
-            'ignoreLevels' => [
-                'debug',
-                'info',
-            ],
-            'ignoreAllInGroup' => [
-                'DBPerformance',
-                'objectcache',// ideally want to show objectcache errors, but not warnings
-            ],
-            'logAllInGroup' => [
-                'WBSTACK',
-                'HttpError',
-                'SpamBlacklistHit',
-                'security',
-                'exception-json',
-                //'error',
-                'fatal',
-                'badpass',
-                'badpass-priv',
-                'api-warning',
-            ],
-            'logAllInGroupExceptDebug' => [
```

Comment:
I believe this may now be ignored when before it wasn't. This being info level log lines explaining that a user was on a IP block list (which we want to log so we can tell users what to do when they complain)